bin/handlemail

```diff
@@ -180,20 +181,43 @@ sub handle_non_bounce_to_null_address {
     # Send an automatic response
     print_log('info', "Received non-bounce to null address, auto-replying");
 
+    my ( $cobrand, $from_addr, $from_name ) = get_config_for_autoresponse();
+
     my $template = path(FixMyStreet->path_to("templates", "email", $cobrand, 'reply-autoresponse'))->slurp_utf8;
 
     # We generate this as a bounce.
     my ($rp) = $data{return_path} =~ /^\s*<(.*)>\s*$/;
     my $mail = FixMyStreet::Email::construct_email({
         'Auto-Submitted' => 'auto-replied',
-        From => [ FixMyStreet->config('CONTACT_EMAIL'),
-                  FixMyStreet->config('CONTACT_NAME') ],
+        From => [ $from_addr, $from_name ],
         To => $rp,
         _body_ => $template,
     });
     send_mail($mail, $rp);
 }
 
+# Based on the address the incoming message was sent to, we might want to
+# use a cobrand's own reply-autoresponse template.
+sub get_config_for_autoresponse {
+    # cobrand might have been set from command line, so prefer that if so.
+    if ( defined $cobrand ) {
+        return ( $cobrand, FixMyStreet->config('CONTACT_EMAIL'), FixMyStreet->config('CONTACT_NAME') );
+    }
+
+    # Try and find a matching email address in the COBRAND_FEATURES config
+    my $recipient = mySociety::HandleMail::get_bounce_recipient($data{message})->address;
+    my $cobrands = FixMyStreet->config('COBRAND_FEATURES')->{do_not_reply_email} || {};
```

Comment:
This will die with an error if COBRAND_FEATURES doesn't exist (and so returns undef), would be good to be careful given it's a script run on email input.